src/plugins/es.json

```diff
@@ -1,98 +1,98 @@
 {
-    "startAgain": "Start Again",
-    "swtchLang": "Français",
-    "onThisPage": "On this page",
-    "htmlCode": "en-CA",
-    "saveButton": "Save",
-    "appTitle": "Algorithmic Impact Assessment",
-    "linkProjectText": "Link to GitHub project repository",
+    "startAgain": "Comenzar de nuevo",
+    "swtchLang": "Español",
+    "onThisPage": "En esta página",
+    "htmlCode": "es-UY",
+    "saveButton": "Guardar",
+    "appTitle": "Evaluación de Impacto Algorítmico",
+    "linkProjectText": "Enlace al repositorio del proyecto GitHub",
```

Comment:
Liga al repositorio del proyecto en GitHub

Reply:
All the recomendation are done!

Reply:
Awesome, thanks! Will be merging shortly! :D